.github/workflows/ci.yml

```diff
@@ -67,4 +67,5 @@ jobs:
         working-directory: ./android
     steps:
       - uses: actions/checkout@v2
+      - run: ./gradlew test -b capacitor/build.gradle -Pandroid.useAndroidX=true -Pandroid.enableJetifier=true
```

Comment:
Instead, please add `test` to the scripts in the line below as such:

```
./gradlew clean build test -b capacitor/build.gradle -Pandroid.useAndroidX=true -Pandroid.enableJetifier=true
```

Reply:
Good spot, done, thanks!